src/main/res/values/strings.xml

```diff
@@ -45,23 +45,23 @@
   <string name="downloading_format">Downloading %s</string>
   <string name="edit_repository">Edit repository</string>
   <string name="empty_search_summary">No matching applications found</string>
-  <string name="error_downloading_format">Error downloading %s</string>
-  <string name="error_syncing_format">Error syncing %s</string>
-  <string name="error_validating_format">Error validating %s</string>
+  <string name="error_downloading_format">Could not download %s</string>
+  <string name="error_syncing_format">Could not sync %s</string>
+  <string name="error_validating_format">Could not validate %s</string>
   <string name="fingerprint">Fingerprint</string>
-  <string name="finished_downloading_format">Finished downloading %s</string>
+  <string name="finished_downloading_format">%s downloaded</string>
   <string name="http_error_description">Invalid server response.</string>
   <string name="http_proxy">HTTP proxy</string>
-  <string name="ignore_all_updates">Ignore all updates</string>
-  <string name="ignore_this_update">Ignore this update</string>
+  <string name="ignore_all_updates">Ignore all future versions</string>
+  <string name="ignore_this_update">Ignore this</string>
   <string name="incompatible_features_description">Missing features:</string>
-  <string name="incompatible_older_description">This version is older than installed on your device.
-    You\'ll have to uninstall the application to proceed.</string>
+  <string name="incompatible_older_description">Older than the version installed on your device.
+    You\'ll have to uninstall the application to install it.</string>
```

Comment:
```suggestion
    Uninstall the current application to install the older version.</string>
```

Reply:
"current" application doesn't sound good. What is "current" application, if user actually wants to install the same "application",  but older "version"?